README.md

```diff
@@ -1,59 +1,96 @@
-# This is a social media website(i-book)
+# 📖 i-Book
 
-- This is just normal project where I have tried cloning a social media website and adding some new ideas.
-- I have thought of making the user anonymous in his/her friends circle and posting and enjoying.
-- This will also feature uploading photos videos and locations with friends
-- You can form groups as well as see what is trending around you
+## 📜 Project Description
 
-![Screenshot_1](https://user-images.githubusercontent.com/78680046/197323291-7bbfd05b-1398-4ff4-8a1b-e4dbb4bbf823.png)
+**i-Book** is a modern social media platform that enables users to connect and interact anonymously with their friends and communities. This project aims to clone a social media website while introducing unique features that prioritize user privacy. Users can upload photos, videos, and locations, form groups, and engage with trending content in their area.
 
+![Screenshot](https://user-images.githubusercontent.com/78680046/197323291-7bbfd05b-1398-4ff4-8a1b-e4dbb4bbf823.png)
 
-# To contribute follow the followings guidelines.
+## 📚 Table of Contents
 
-## To access this fork and then clone it:-
+1. [Installation Guide](#installation-guide)
+2. [Features](#features)
+3. [Contributing](#contributing)
+4. [Acknowledgements](#acknowledgements)
 
-- Fork it and star it , then clone with <code> git clone <repo_link> </code> command.
-- Make your new branch with your username easy to identifiable. Use command <code> git branch <your_branch_name></code> and then <code> git checkout <your_branch_name></code>.
-- Just update the UI of the code which is in client folder, no need to change anything in the API folder (much appreciated).
-- <code>cd client</code>
-- <code> npm install </code>
-- Create ```.env``` file in the client folder and add these lines
-- <code>REACT_APP_PUBLIC_FOLDER=http://localhost:8000/images/
-    GENERATE_SOURCEMAP=false
-    PATH=http://localhost:8000
-    REACT_APP_PATH_TO_BACKEND=http://localhost:8000
-  </code>
-- Then <code>npm run start</code> to start working on the frontend.
-- <code>cd api</code> in new terminal.
-- <code> npm install </code>
-- Then <code>npm run start</code> to start backend.
+## ⚙️ Installation Guide
 
+To get started with the **i-Book** project, follow these steps:
 
-# This project also support docker development
+### 1. Clone the Repository
+Clone the project repository using:
+```bash
+git clone <repo_link>
+```
 
-## To start both the frontend and backend service at the same time
+### 2. Navigate to the Client Directory
+```bash
+cd client
+```
 
-- <code>docker-compose up</code>
+### 3. Install Dependencies
+```bash
+npm install
+```
 
-## To start the server side development locally in docker container
+### 4. Create an Environment File
+Create a `.env` file in the `client` directory and add the following lines:
+```env
+REACT_APP_PUBLIC_FOLDER=http://localhost:8000/images/
+GENERATE_SOURCEMAP=false
+PATH=http://localhost:8000
```

Comment:
Could you please remove this line. This was added by mistake. 
Thanks

Reply:
@JollyJolli

Reply:
Yes of course!

Reply:
What do i put?

Reply:
nothing just need to remove that part

Reply:
All the ENV or only the localhost?

Reply:
only this line "PATH=http://localhost:3000", all other ENV var should remain

Reply:
Just commit the changed of removing the PATH